packages/core/src/app/core/state/page.state.ts

```diff
@@ -1,5 +1,5 @@
 import { State, Action, StateContext, Selector } from '@ngxs/store';
-import { CodeGenKind, XlayersExporterNavBar, XlayersNgxEditorModel } from 'src/app/editor/code-editor/editor-container/codegen/codegen.service';
+import { CodeGenKind, XlayersExporterNavBar, XlayersNgxEditorModel } from '../../editor/code-editor/editor-container/codegen/codegen.service';
```

Comment:
why not use the TS path aliases as referenced in the tsconfig?

MOTE: This also applied to the rest of the files in this PR.

Reply:
I had some problems with Jest which can't import the module with an absolute path with this file structure particularly.

But yes i agree that can be a little weird.